libretroshare/src/pqi/pqinetwork.cc

```diff
@@ -317,7 +322,15 @@ bool getLocalAddresses(std::list<sockaddr_storage> & addrs)
 		}
 	}
 	free(adapter_addresses);
-#else // WINDOWS_SYS
+#elif defined(__ANDROID__)
+	foreach(QHostAddress qAddr, QNetworkInterface::allAddresses())
```

Comment:
The policy of not using Qt in libretroshare should be enforced. Can you please find a way to fix this?